### DISCUSSION TITLE
v1.3.2

### DISCUSSION THREAD

**Discussion by LuckyDucko** [LuckyDucko/Mopups#142] [Announcements]
Hi All

Apologies for being absent regarding this project. I haven't been involved at all in mobile development with Maui, i do apologise for that. I am glad that some people from the community are able to come in and help sort things out for each other. 

## What's Changed
* Check for api level when checking for animations by @eichbaum in https://github.com/LuckyDucko/Mopups/pull/109
* Fix: Windows window resizing by @IeuanWalker in https://github.com/LuckyDucko/Mopups/pull/104
* Fix "VirtualView cannot be null here" on Android when `OnLayout` is fired after the handler is disconnected by @romerotg in https://github.com/LuckyDucko/Mopups/pull/107
* iOS: Popup is closing when background is touched instead of tapped by @s153625 in https://github.com/LuckyDucko/Mopups/pull/108
* Fix: Accessibility with overlaying popups by @IeuanWalker in https://github.com/LuckyDucko/Mopups/pull/136
* Fix: Exception "MauiContext should have been set on parent" when removing a child from its parent layout on iOS by @troy-ob in https://github.com/LuckyDucko/Mopups/pull/115
* Open popups on the main application window by @torfluor in https://github.com/LuckyDucko/Mopups/pull/125
* Use interfaces instead of implementations  by @bdovaz in https://github.com/LuckyDucko/Mopups/pull/139
* Null VirtualView by @IeuanWalker in https://github.com/LuckyDucko/Mopups/pull/137

## New Contributors
* @romerotg made their first contribution in https://github.com/LuckyDucko/Mopups/pull/107
* @s153625 made their first contribution in https://github.com/LuckyDucko/Mopups/pull/108
* @troy-ob made their first contribution in https://github.com/LuckyDucko/Mopups/pull/115
* @torfluor made their first contribution in https://github.com/LuckyDucko/Mopups/pull/125
* @bdovaz made their first contribution in https://github.com/LuckyDucko/Mopups/pull/139

**Full Changelog**: https://github.com/LuckyDucko/Mopups/compare/1.3.0...v1.3.2

<hr /><em>This discussion was created from the release <a href='https://github.com/LuckyDucko/Mopups/releases/tag/v1.3.2'>v1.3.2</a>.</em>

**Comment by nguyendinhchinh** [comment]
I took v1.3.2 and tested it again that the problem with https://github.com/LuckyDucko/Mopups/issues/95 had not been solved
@LuckyDucko You can trigger the new Nuget version to fix this problem (https://github.com/LuckyDucko/Mopups/issues/95) soon?

This is my temporary solution
Reference: https://github.com/LuckyDucko/Mopups/pull/108#issuecomment-2667721549

![image](https://github.com/user-attachments/assets/76f6fdc8-5ac9-4f79-bd18-787a5f119066)
